### PR TITLE
adding arista_eos_show_interfaces_transceiver.template and edit arist…

### PR DESCRIPTION
**Pull request by Sandeepsr**
…a_eos_show_ip_bgp_summary.template for eos 4.19.1

adding arista_eos_show_interfaces_transceiver.template and edit arista_eos_show_ip_bgp_summary.template for eos 4.19.1

##### ISSUE TYPE

 - New Template Pull Request
   arista_eos_show_interfaces_transceiver.template
-Update to template
   arista_eos_show_ip_bgp_summary.template

##### COMPONENT

arista_eos_show_interfaces_transceiver.template
arista_eos_show_ip_bgp_summary.template
##### SUMMARY

adding new arista_eos_show_interfaces_transceiver.template and edit arista_eos_show_ip_bgp_summary.template for eos 4.19.1 support 

```

```

### CODE REVIEW

**Comment by Sandeepsr** [comment]
@itdependsnetworks , can you check this now, have I have updated the raw files as well.

**Comment by itdependsnetworks** [comment]
@Sandeepsr Can you add the test rather than replace? These look subtly different and I would like to make sure we support both moving forward

**Comment by Sandeepsr** [comment]
@itdependsnetworks , can new tests be in same folder as existing one ?

**Comment by Sandeepsr** [comment]
@itdependsnetworks , added new test files.

File: templates/arista_eos_show_interfaces_transceiver.template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Value PORT (\S+\d+\S+)
```
Comment (itdependsnetworks) [inline templates/arista_eos_show_interfaces_transceiver.template]: Could it not be `Eth3` as an example?

File: templates/arista_eos_show_ip_bgp_summary.template [hunk 2]
```diff
@@ -6,12 +6,15 @@ Value MSG_RCVD (\d+)
 Value MSG_SENT (\d+)
 Value IN_QUEUE (\d+)
 Value OUT_QUEUE (\d+)
-Value UP_DOWN (\d+:\d+:\d+|\d+)
+Value UP_DOWN (\d+\S\d+\S|\d+:\d+:\d+)
```
Comment (itdependsnetworks) [inline templates/arista_eos_show_ip_bgp_summary.template]: seems too complex, what happens when under an hour or under a minute? I'm ok with being looser on the capture groups and if the whole line is accounted for.

File: tests/arista_eos/show_interfaces_transceiver/arista_eos_show_interfaces_transceiver.raw [hunk 3]
```diff
@@ -0,0 +1,12 @@
+If device is externally calibrated, only calibrated values are printed.
+N/A: not applicable, Tx: transmit, Rx: receive.
+mA: milliamperes, dBm: decibels (milliwatts).
+                               Bias      Optical   Optical                
+          Temp       Voltage   Current   Tx Power  Rx Power               
+Port      (Celsius)  (Volts)   (mA)      (dBm)     (dBm)     Last Update  
+-----     ---------  --------  --------  --------  --------  -------------------
```
Comment (itdependsnetworks) [inline tests/arista_eos/show_interfaces_transceiver/arista_eos_show_interfaces_transceiver.raw]: Let's do a state change here, and be much looser in the actual capture groups. 

something like (not tested) e.g.

```
Value PORT (\S+)
Value TEMP (\S+)
Value VOLTAGE (\S+)
Value BIAS_CURRENT (\S+)
Value TX_POWER (\S+)
Value RX_POWER (\S+)


Start
  ^---- > Begin

Begin
 ^${PORT}\s+${TEMP}\s+${VOLTAGE}\s+${BIAS_CURRENT}\s+${TX_POWER}\s+${RX_POWER}\s+(\S+ ago|N/A) -> Record

```

Comment (Sandeepsr) [reply tests/arista_eos/show_interfaces_transceiver/arista_eos_show_interfaces_transceiver.raw]: @itdependsnetworks , thanks for review. I was on vacations and could not get this done earlier . Please review suggested changes and approve as appropriate

**Comment by Sandeepsr** [comment]
@itdependsnetworks  , please review now

**Pull request merged**